front/app/components/Areas/FollowArea/UpdateFollowAreaWithProjects.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Since this is only used in the `Areas` widget, I think it should live there.  I would move it to the `front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/Areas` folder

Reply:
I've put `UpdateFollowAreaWithProjects` in there because `UpdateFollowArea` was already there. I like that they're together with `BaseUpdateFollowArea` so all variations can be seen in one view. And I think they're generic enough not to be in a specific feature's folder. Although they're currently not getting reused so I can see your point.

Reply:
Feel free to make a PR to take out both and put them closer to where they're used if you feel strongly about it and think they won't be reused elsewhere.